tests/test-cases/demo-vendoring.yaml

```
@@ -39,3 +39,60 @@ tests:
+    workdir: "fixtures/scenarios/vendor-creds-sanitize"
+    command: "atmos"
+    args:
+      - "vendor"
+      - "pull"
+      - "--logs-level=Debug"
+    env:
+      ATMOS_GITHUB_TOKEN: supersecret
+    expect:
+      diff: []
+    stdout:
+      - "!supersecret"  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)